web/src/components/inputs/input.tsx

```diff
@@ -280,35 +280,44 @@ export const PasswordField = ({
           {label} {required && <span className="text-gray-500">*</span>}
         </label>
       )}
-      <Field name={name} defaultValue={defaultValue}>
-        {({
-          field,
-          meta
-        }: FieldProps) => (
-          <div className="sm:col-span-2 relative">
-            <input
-              {...field}
-              id={name}
-              type={isVisible ? "text" : "password"}
-              autoComplete={autoComplete}
-              className={classNames(meta.touched && meta.error ? "focus:ring-red-500 focus:border-red-500 border-red-500" : "focus:ring-blue-500 dark:focus:ring-blue-500 focus:border-blue-500 dark:focus:border-blue-500 border-gray-300 dark:border-gray-700", "mt-2 block w-full dark:bg-gray-800 dark:text-gray-100 rounded-md")}
-              placeholder={placeholder}
-            />
-
-            <div className="absolute inset-y-0 right-0 px-3 flex items-center" onClick={toggleVisibility}>
-              {!isVisible ? <EyeIcon className="h-5 w-5 text-gray-400 hover:text-gray-500" aria-hidden="true" /> : <EyeSlashIcon className="h-5 w-5 text-gray-400 hover:text-gray-500" aria-hidden="true" />}
-            </div>
+      <div className="relative">
```

Comment:
Pretty sure you don't need this parent relative here.

Reply:
True. fixed. Thanks for looking over it! Always a pleasure!